src/main/java/org/cbioportal/service/ClinicalDataDensityPlotService.java

```diff
@@ -0,0 +1,23 @@
+package org.cbioportal.service;
+
+import org.cbioportal.model.ClinicalData;
+import org.cbioportal.model.DensityPlotData;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public interface ClinicalDataDensityPlotService {
+    DensityPlotData getDensityPlotData(List<ClinicalData> filteredClinicalData,
```

Comment:
Can we Take these arguments and throw them into a POJO (maybe using a builder pattern) so we aren't passing so many  args to the fn

Reply:
good idea